src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/ClosePaymentRequestBuilder.kt

```diff
@@ -0,0 +1,3 @@
+package it.pagopa.ecommerce.eventdispatcher.services.v2
+
+class ClosePaymentRequestBuilder {}
```

Comment:
can we remove this class? seems to be unused

Reply:
Fixed in 2e391fb